### PR TITLE
Both cloud and standalone readmes

### PR DESCRIPTION
**Pull request by squaremo**
This adapts the files in `deploy/` to cover two cases: just using the Weave Cloud service, and running Flux standalone.

The directory `testdata/` is removed as part of this, since those are now in their own repository, which is easier (or less confusing) to fork while following the instructions given here.

### CODE REVIEW

**Comment by rade** [comment]
so does this *fix* #234?

**Comment by squaremo** [comment]
Yes. Fixes #234.

File: deploy/README.md [hunk 1]
```diff
@@ -1,31 +1,39 @@
...
- locally, for example, you can do
+There's two ways to deploy Flux: you can use the hosted service at
+Weave Cloud, or you can run the service yourself.
 
-```
-eval $(minikube docker-env)
-make clean all
-```
+Using Weave Cloud is described in [cloud/](./cloud/README.md); running
+it all yourself is described in [standalone/](README.md).
```
Comment (paulbellamy) [inline deploy/README.md]: Is this standalone link right? It is different from the cloud one...

File: deploy/README.md [hunk 2]
```diff
@@ -1,31 +1,39 @@
...
+repository to your own account on github.
+
+You can run the helloworld service by creating the deployment and
+service resources given as files in that repo:
 
-Fork the Flux repository on github (you may also wish to rename it,
-e.g., to `flux-testdata`). Now, we're going to add a deploy key so
-Flux can push to that repo. Generate a key in the console:
+```
+flux-example$ kubectl create -f helloworld-deploy.yaml -f helloworld-svc.yaml
```
Comment (paulbellamy) [inline deploy/README.md]: might be more clear to do it this way, not sure...
```
$ cd flux-example
$ kubectl create -f helloworld-deploy.yaml -f helloworld-svc.yaml
```

File: deploy/README.md [hunk 3]
```diff
@@ -104,8 +81,8 @@ Here's an example with values filled in:
 
 ```yaml
 git:
-  URL: git@github.com:squaremo/flux-testdata
-  path: testdata
+  URL: git@github.com:squaremo/flux-example
```
Comment (paulbellamy) [inline deploy/README.md]: nice repo name. do you mean weaveworks?

File: deploy/README.md [hunk 4]
```diff
@@ -39,45 +47,14 @@ On the Github page for your forked repo, go to the settings and find
...
-
 ## Uploading a configuration
 
 To begin using Flux, you need to provide at least the git repository
-and the key from earlier.
+and the key from earlier. The following assumes you have set the
+environment variable `FLUX_SERVICE_TOKEN` if you are using Weave
+Cloud, or `FLUX_URL` if you are using a standalone deployment of Flux;
+add the `--token` or `--url` argument in, if you prefer to supply
+those explicitly.
```
Comment (paulbellamy) [inline deploy/README.md]: Link to instructions about these?

File: deploy/standalone/flux-deployment.yaml [hunk 5]
```diff
@@ -11,14 +11,11 @@ spec:
...
-        image: weaveworks/fluxd
-        imagePullPolicy: Never # must build manually
+        image: quay.io/weaveworks/fluxd:master-6cc08e4
         args:
-        - fluxd
         - --fluxsvc-address=ws://localhost:3030
       - name: fluxsvc
-        image: weaveworks/fluxsvc
+        image: quay.io/weaveworks/fluxsvc:master-6cc08e4
         imagePullPolicy: Never
```
Comment (paulbellamy) [inline deploy/standalone/flux-deployment.yaml]: never pull?

File: deploy/README.md (hunk 3 above)
Comment (squaremo) [inline deploy/README.md]: I do not; I'm giving an example of a fork which I've made for the purpose of trying it out, as suggested above.

File: deploy/standalone/flux-deployment.yaml (hunk 5 above)
Comment (squaremo) [inline deploy/standalone/flux-deployment.yaml]: Good spot, this should be `IfNotPresent` shouldn't it.

File: deploy/README.md (hunk 4 above)
Comment (paulbellamy) [inline deploy/README.md]: Or somewhere else in the readme, whatever.

**Comment by squaremo** [comment]
@paulbellamy Made some changes ..

**Comment by lukemarsden** [comment]
fyi, this broke the katacoda, but there was no way for you to know that. We're fixing it now.

**Pull request merged**